business/istio_config.go

```diff
@@ -133,7 +134,18 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 	go func(errChan chan error) {
 		defer wg.Done()
 		if criteria.IncludeGateways {
-			if gg, ggErr := in.k8s.GetGateways(criteria.Namespace); ggErr == nil {
+			var gg []kubernetes.IstioObject
+			var ggErr error
+			// Check if namespace is cached
+			if kialiCache != nil && kialiCache.CheckIstioResource("Gateway") && kialiCache.CheckNamespace(criteria.Namespace) {
```

Comment:
This is just nitpicking (applies to many rows here), but instead of rewriting these "Gateway", maybe the types.go's defined variables could have been used - not that I expect them to ever change.

Reply:
Yeah, it was in my todo list to move this strings to constants.
I can do it in the PR.